### PR TITLE
fix some bug about Request review

### PR DESCRIPTION
**Pull request by a1012112796**
* fix ``CreateComment`` wrong using ,it will not work when use Sqlite
* fix wrong js click event code , it will send wrong data when it has many choices
* fix wrong conditions check in initIssueComments after #10972

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: web_src/js/index.js [hunk 1]
```diff
@@ -663,15 +663,19 @@ function initInstall() {
 function initIssueComments() {
   if ($('.repository.view.issue .comments').length === 0) return;
 
-  $('.re-request-review').click((event) => {
-    const $this = $('.re-request-review');
+  $('.re-request-review').on('click', function (event) {
+    const url = $(this).data('update-url');
+    const issue_id = $(this).data('issue-id');
```
Comment (lafriks) [inline web_src/js/index.js]:
```suggestion
    const issueId = $(this).data('issue-id');
```

File: web_src/js/index.js [hunk 2]
```diff
@@ -663,15 +663,19 @@ function initInstall() {
...
+    const id = $(this).data('id');
+    const is_checked = $(this).data('is-checked');
```
Comment (lafriks) [inline web_src/js/index.js]:
```suggestion
    const isChecked = $(this).data('is-checked');
```

File: web_src/js/index.js [hunk 3]
```diff
@@ -663,15 +663,19 @@ function initInstall() {
...
+
     event.preventDefault();
     updateIssuesMeta(
-      $this.data('update-url'),
+      url,
       '',
-      $this.data('issue-id'),
-      $this.data('id'),
-      $this.data('is-checked')
+      issue_id,
```
Comment (lafriks) [inline web_src/js/index.js]:
```suggestion
      issueId,
```

File: web_src/js/index.js [hunk 4]
```diff
@@ -663,15 +663,19 @@ function initInstall() {
...
+      id,
+      is_checked
```
Comment (lafriks) [inline web_src/js/index.js]:
```suggestion
      isChecked
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

File: models/review.go [hunk 5]
```diff
@@ -575,7 +575,7 @@ func RemoveRewiewRequest(issue *Issue, reviewer *User, doer *User) (comment *Com
 		return nil, err
 	}
 
-	comment, err = CreateComment(&CreateCommentOptions{
+	comment, err = createComment(sess, &CreateCommentOptions{
```
Comment (6543) [inline models/review.go:582]: nice catch

**Comment by 6543** [comment]
@a1012112796 `GetReviewerByIssueIDAndUserID` also need a version witch can use sess !!

**Comment by 6543** [comment]
for **GetReviewerByIssueIDAndUserID**: rename it to **getReviewerByIssueIDAndUserID** and create a second function caled **GetReviewerByIssueIDAndUserID** witch use **getReviewerByIssueIDAndUserID**

similar to what you can find here:
https://github.com/go-gitea/gitea/blob/bc362ea3c611a94e7920a3cf4691fd108f874ef0/models/issue_comment.go#L768-L773

**Comment by 6543** [comment]
then use getReviewerByIssueIDAndUserID in RemoveRewiewRequest - thanks!

**Comment by a1012112796** [comment]
@6543 done your suggestion, please review . Thanks

**Review by guillep2k — APPROVED** [review]

**Comment by 6543** [comment]
@a1012112796 thanks, it was exactly what I ment :)

**Pull request merged**